### PR TITLE
TDL-5992 modify subscription items stream

### PR DESCRIPTION
**Pull request by namrata270998**
# Description of change
- Added the fields in `KNOWN_MISSING_FIELDS` for subscription_items stream to the schema
- Removed the fields from `KNOWN_MISSING_FIELDS`.

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: tests/test_all_fields.py [hunk 1]
```diff
@@ -34,8 +34,6 @@
     'payouts':set(),
     'charges': set(),
     'subscription_items':{
-        'tax_rates',
-        'price',
         'updated',
```
Comment (namrata270998) [inline tests/test_all_fields.py:37]: `tax_rates` and `price` - both fields added to schema. Now it replicates in actual records also. So, removed it from the `KNOWN_MISSING_FIELDS` map.

File: tests/base.py [hunk 2]
```diff
@@ -391,7 +391,7 @@ def records_data_type_conversions(self, records):
             ]
             object_keys = [
                 'discount', 'plan', 'coupon', 'status_transitions', 'period', 'sources', 'source',
-                'package_dimensions',
+                'package_dimensions','price'
```
Comment (karanpanchal-crest) [inline tests/base.py:394]: Why did we add 'price' in this variable?

File: tests/test_all_fields.py (hunk 1 above)
Comment (karanpanchal-crest) [reply tests/test_all_fields.py:37]: @namrata270998 Please update the PR code changes as per the changes incorporated in 'invoice_line_items' changes - https://github.com/singer-io/tap-stripe/pull/112/files#diff-eeff454edf0e2c75012eb431c28a488069dff76be419b0a1396fc2b451c4d156L51

File: tap_stripe/schemas/subscription_items.json [hunk 3]
```diff
@@ -26,6 +26,296 @@
       "format": "date-time"
```
Comment (dbshah1212) [inline tap_stripe/schemas/subscription_items.json:26]: Can we add a reference document link?

**Review by karanpanchal-crest — CHANGES_REQUESTED** [review]

File: tap_stripe/schemas/subscription_items.json (hunk 3 above)
Comment (namrata270998) [reply tap_stripe/schemas/subscription_items.json:26]: added the doc link in the comment 
https://github.com/singer-io/tap-stripe/pull/114#discussion_r777866475

File: tests/test_all_fields.py (hunk 1 above)
Comment (namrata270998) [reply tests/test_all_fields.py]: Incorporated the changes as suggested

File: tests/base.py (hunk 2 above)
Comment (namrata270998) [reply tests/base.py:394]: The price variable had a `created` field which had the epoch fromat. This function is to convert the epoch time to datetime for the fields mentioned in the list `timestamp_to_datetime_keys` which are present as nested fields in the `object_keys`.
Also added a code comment for the same.

**Review by karanpanchal-crest — APPROVED** [review]

**Review by KrisPersonal — APPROVED** [review]

**Review by kspeer825 — APPROVED** [review]

**Pull request merged**